scripts/clang-format.py

```diff
@@ -14,7 +14,7 @@
     'lib',
     'master',
 ]
-husky_root = os.getenv('HUSKY_ROOT', '.')
+husky_root = '.';
```

Comment:
The second parameter in `getenv` is the default value to return if `HUSKY_ROOT` doesn't exist. This has considered your situation.